router-tests/ratelimit_test.go

```diff
@@ -675,16 +675,133 @@ func TestRateLimit(t *testing.T) {
 	})
 	t.Run("Cluster Mode", func(t *testing.T) {
 		var (
-			clusterUrlSlice = []string{"localhost:7000", "localhost:7001", "localhost:7002"}
-			password        = "test"
+			clusterUrlSlice     = []string{"redis://cosmo:test@localhost:7000", "redis://cosmo:test@localhost:7001", "redis://cosmo:test@localhost:7002"}
+			noSchemeClusterUrls = []string{"localhost:7000", "localhost:7001", "localhost:7002"}
+			user                = "cosmo"
+			password            = "test"
 		)
 
+		t.Run("cluster url ordering", func(t *testing.T) {
```

Comment:
I could be wrong, but it looks like this could be a table test instead of 3x the main test code, if not a tdt, perhaps a "test runner" function?